src/main.py

```
@@ -71,8 +73,10 @@ async def setup_admin_jwt():
     def before_send(event: "Event", hint: "Hint") -> "Event | None":
         if "exc_info" in hint:
             _, exc_value, _ = hint["exc_info"]
-            # Filter out HonchoExceptions from being sent to Sentry
-            if isinstance(exc_value, HonchoException):
+            # Filter out exceptions that shouldn't be sent to Sentry
+            if isinstance(
+                exc_value, HonchoException | ValidationError | RequestValidationError
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)